### DISCUSSION TITLE
WIX Toolset 3.10 and 3.11.2: light.exe

### DISCUSSION THREAD

**Discussion by jackandjoeatthecomputer** [wixtoolset/issues#6955] [Questions]
I run my first WIX project. Simple wxs file with one component and one file. 
<?xml version='1.0' encoding='utf-8'?>
<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi'>
<Product Name='Foobar 1.0' Manufacturer='Acme Ltd.'
        Id='E45C8A57-FA6B-492F-A5AC-FCC8545DE419' 
        UpgradeCode='716019A7-E215-4253-AA0F-910A2BD09D73'
        Language='2055' Codepage='1141' Version='1.0.0'>
<Package Id='*' Keywords='Installer' Description="Susannes Rezeptbuch 1.0 Installer"
        Comments='Susannes Rezeptbuch ist ein Rezepteverwaltungstool' Manufacturer='Susanne Krause'
        InstallerVersion='100' Languages='2055' Compressed='yes' SummaryCodepage='1141' />
<Media Id="1" Cabinet="media1.cab" EmbedCab="yes" />
<Icon Id="Rezeptbuch_icon" SourceFile="susannesrezeptbuch.ico" />
<Directory Id='TARGETDIR' Name='SourceDir'>
	<Directory Id='ProgramFilesFolder' Name='PFiles'>
    		<Directory Id='SusannesRezeptbuch' Name='SusannesRezeptbuch'>
        		<Directory Id='INSTALLDIR' Name='Susannes Rezeptbuch 1.0'>
        			<Component Id='MainExecutable' Guid='E7011577-EE28-41D9-8F4F-2C924BE1F1E0'>
        				<File Id='SusannesRezeptbuch.bat' Name='SusannesRezeptbuch.bat' DiskId='1' Source='SusannesRezeptbuch.bat' KeyPath='yes'>
        				    <Shortcut Id="startmenuRezeptbuch10" Directory="ProgramMenuDir" Name="Susannes Rezeptbuch 1.0"
						            WorkingDirectory='INSTALLDIR' Icon="Rezeptbuch_icon" IconIndex="0" Advertise="yes" />
							<Shortcut Id="desktopRezeptbuch10" Directory="DesktopFolder" Name="Susannes Rezeptbuch 1.0"
						            WorkingDirectory='INSTALLDIR' Icon="Rezeptbuch_icon" IconIndex="0" Advertise="yes" />
					</File>
				</Component>
			</Directory>
		</Directory>
	</Directory>
	<Directory Id="ProgramMenuFolder" Name="Programs">
		<Directory Id="ProgramMenuDir" Name="Foobar 1.0">
	            <Component Id="ProgramMenuDir" Guid="BE847EEE-8AF3-41E5-B422-60534B0977C1">
	                <RemoveFolder Id='ProgramMenuDir' On='uninstall' />
	                <RegistryValue Root='HKCU' Key='Software\Susanne Krause\Susannes Rezeptbuch' Type='string' Value='' KeyPath='yes' />
	            </Component>
	        </Directory>
	</Directory>	
	<Directory Id="DesktopFolder" Name="Desktop" />
</Directory>
      <Feature Id="DefaultFeature" Level="1">
         <ComponentRef Id="MainExecutable"/>
      </Feature>
</Product>
</Wix>

running candle SusannesRezeptbuch.wxs vom commandline works fine. 

light.exe hangs:

C:\Users\baerbel\Rezeptbuch_compiledir\2022.0.1_commandline-WIX>light.exe -v SusannesRezeptbuch.wixobj
Windows Installer XML Toolset Linker version 3.10.4.4718
Copyright (c) .NET Foundation and contributors. All rights reserved.

Updating file information.
Creating cabinet files.
There will be '4' threads used to produce CAB files.
Creating cabinet 'C:\Users\baerbel\AppData\Local\Temp\vpztky12\#media1.cab'.
Generating database.

It hangs and dont ends. After 10 minutes I interrupt it.

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh]. You might also consider asking your question on the [wix-users mailing list][ml] or posting on [Stack Overflow][so].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
[ml]: http://wixtoolset.org/documentation/mailinglist/
[so]: https://stackoverflow.com/questions/tagged/wix

**Comment by robmen** [comment]
Try attaching the debugger and getting the callstack. That might provide more insight as where to start looking.

**Comment by jackandjoeatthecomputer** [comment]
Thank you

Rob Mensching ***@***.***> schrieb am Do., 10. Nov. 2022,
07:53:
